grading/Sample_CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8)
+cmake_minimum_required (VERSION 3.5)
```

Comment:
This should be fine(?) since this is a relatively old version of CMake so it should be on all recent servers.
This is used when building any autograding assignment, so if we don't have this version of CMake on the server, this would be a big problem.